README.md

```diff
@@ -51,6 +51,33 @@ export default defineNuxtConfig({
 
 That's it! You can start writing and rendering markdown files in your Nuxt project ✨
 
+### WebAssembly (WASM)
+
+For some Nuxt projects, depending on your build preset, you may need to adjust settings to allow WebAssembly (WASM) to be used for the default Shiki highlighter.
+
+```ts [nuxt.config.ts]
+export default defineNuxtConfig({
+  modules: ['@nuxtjs/mdc'],
+  $production: {
+    nitro: {
+      // Enable the wasm feature in production for the Shiki syntax highlighter
+      experimental: {
+        wasm: true,
```

Comment:
FYI, mdc module already has built-in logic to add wasm related logic. it should not be required unless that current mdc logic is outdated.

Reply:
It doesn't seem to be handled properly (?) in a Nuxt project deployed to Cloudflare Pages without this setting. 

If you check the reproduction deploys [in this comment](https://github.com/nuxt-modules/mdc/issues/159#issuecomment-1985785385) you'll see two things are missing:

1. The `nitropack` version needed to be bumped to `2.9.x`
2. Without the `nitro.experimental.wasm: true` setting, the syntax highlighting is still not functional.

Let me know if you want me to update one of the reproduction repos/branches?

Reply:
https://github.com/nuxt-modules/mdc/blob/420a5026c022c4ed2dfe19f8ca2f2d1cbce6c759/src/module.ts#L149
The way I'm reading this line, I believe the setting is required? Feel free to correct me 😅

Reply:
@pi0 I modified my local `@nuxtjs/mdc` build and changed the line referenced above to this (removing the check for the `nitro.experimental.wasm: true` setting):

```ts
useWasmAssets: !nuxt.options.dev
```

If I remove the line, I can then remove the `nitro.experimental.wasm: true` in my host project, and things work as expected.